modules/rdns/controllers/service/handler.go

```diff
@@ -183,7 +183,7 @@ func (h *handler) staticAddress() ([]adminv1.Address, error) {
 func (h *handler) getAddresses(svc *corev1.Service) ([]adminv1.Address, bool, error) {
 	result, err := h.staticAddress()
 	if len(result) > 0 || err != nil {
-		return result, false, err
+		return result, true, err
```

Comment:
this is ok ?

Reply:
This is the fix for RKE bug, when IP-address is set nodeport must be returned to true.